csp-guru/datapackage.json

```diff
@@ -1,11 +1,161 @@
     "resources": [
         {
-            "path": "csp-guru.csv",
-            "hash": "0ed89d2fe7c96615ad70d79ce64693a8"
+            "path": "CSP.guru-2016-08-30.csv",
+            "hash": "0ed89d2fe7c96615ad70d79ce64693a8",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "Title",
+                        "description": "The name of the CSP plant"
+                    },
+                    {
+                        "name": "Year_operational",
+                        "description": "indicates the year of the first power output"
+                    },
+                    {
+                        "name": "Status",
+                        "description": "Dinstinguishes if a CSP plant is operational or under construction"
+                    },
+                    {
+                        "name": "Country",
+                        "description": "Country where the CSP plant is located"
+                    },
+                    {
+                        "name": "City",
+                        "description": "City or municipality where the CSP plant is located"
+                    },
+                    {
+                        "name": "Lat",
+                        "description": "Latitude of the plant"
+                    },
+                    {
+                        "name": "Lon",
+                        "description": "Longitude of the plant"
+                    },
+                    {
+                        "name": "DNI",
+                        "description": "Direct Normal Irradiance [kWh/m²/y] indicates the solar resources"
+                    },
+                    {
+                        "name": "Year_construction_start",
+                        "description": "indicates the year when construction started"
+                    },
+                    {
+                        "name": "Technology",
+                        "description": "Dinstinguishes CSP designs in Trough, Tower, Hybrid, Fresnel "
+                    },
+                    {
+                        "name": "Capacity",
+                        "description": "Nominal capacity of the turbine [MW]"
+                    },
+                    {
+                        "name": "Storage_capacity",
+                        "description": " how many hours the turbine can run on stored heat alone"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area in square meters"
```

Comment:
I'd be consistent with the formatting of units. As you used "[unit]" in most or all other cases, I'd use it here as well: [m^2].